docs/federated_testing.md

```diff
@@ -1,85 +1,4 @@
 # Federated Testing
```

Comment:
@krzyzacy, there is a slight confusion here. Although the section name say `Federated Testing`, This section has nothing to do with `Cluster federation`. It has more to do with independent test environments submitting their test results to GCS which can be viewed in test-grid.

Reply:
+1

Reply:
ack, I'll rewrite the out-dated part and keep the testgrid bucket section.